Code/GraphMol/MolAlign/testMolAlign.cpp

```diff
@@ -384,8 +398,9 @@ void testMMFFO3AConstraints() {
   TEST_ASSERT(o3a);
   o3a->align();
   delete o3a;
-  double d = (m2.getConformer().getAtomPos(cIdx) -
-              m1->getConformer().getAtomPos(cIdx)).length();
+  double d =
+      (m2.getConformer().getAtomPos(cIdx) - m1->getConformer().getAtomPos(cIdx))
```

Comment:
I format the files using clang-format. There is a setting file for it at the root of the rdkit tree. If you are going to use an automatic formatter, it should be this one. Otherwise please do not automatically format C++ code.
And, yes, I really need to finish the Contributing document that explains all this stuff. :-S